### ISSUE TITLE
Added option to show image in details widget

### ISSUE THREAD

**Issue by cklosowski** [easydigitaldownloads/Easy-Digital-Downloads#4239]
As originally proposed in #4221, this is an issue for it so we can work on it in an issue branch instead of PR, to help with the tests and any further development needed on it.

**Comment by cklosowski** [comment]
Just waiting on a PR against this new issue branch `issue/4239` to clean up the unit tests.
